AutoWorkup/workflows/WorkupComputeLabelVolume.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+from __future__ import absolute_import
+from builtins import str
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'str'](https://www.codacy.com/app/hans-j-johnson/BRAINSTools/file/2627303137/issues/source?bid=3194574&fileBranchId=3316990#l5)